### PR TITLE
Update Schema visualization to v0.3.0

### PR DESCRIPTION
**Pull request by R0tenur**
Updates Schema visualization to v0.3.0

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -2981,13 +2981,13 @@
 					"versions": [
 						{
 							"version": "0.2.0",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:2983]: Update the version here as well

File: extensionsGallery.json [hunk 2]
```diff
@@ -2980,8 +2980,8 @@
 					},
 					"versions": [
 						{
-							"version": "0.2.0",
-							"lastUpdated": "2/2/2020",
+							"version": "0.3.0",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:2983]: Update the link down below as well

**Comment by Charles-Gagnon** [comment]
@R0tenur The gallery has been updated - you should see the new version now

**Comment by R0tenur** [comment]
Thanks!

**Pull request merged**